src/ImageSharp/Formats/Tiff/TiffDecoderCore.cs

```diff
@@ -793,7 +793,7 @@ private int CalculateStripBufferSize(int width, int height, int plane = -1)
             }
         }
 
-        int bytesPerRow = ((width * bitsPerPixel) + 7) / 8;
+        int bytesPerRow = (int)(((uint)(width * bitsPerPixel) + 7) / 8);
```

Comment:
AGain, I'm not sure if the complexity here is worth the effort. It's not in a loop and obfuscates the code.

Reply:
I'll revert these two.